app/controllers/mixins/start_url.rb

```diff
@@ -41,7 +36,6 @@ def start_page_options
 
   def start_page_allowed?(start_page)
     return false if STORAGE_START_PAGES.include?(start_page)
```

Comment:
Looks right, that product setting is gone, so never true, and StorageManagerController still exists, is in the menu (under the `storage_manager_show_list` feature) .. not seeing a problem :)